apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.$projectParam/route.tsx

```diff
@@ -35,9 +36,16 @@ export const loader = async ({ request, params }: LoaderFunctionArgs) => {
 
     telemetry.project.identify({ project });
 
-    return typedjson({
-      project,
-    });
+    const session = await setCurrentProjectId(project.id, request);
```

Comment:
This is where the current projectId is set, so the new sidebar can always show a project